### DISCUSSION TITLE
Use of actInv() in frameClassicAcceleration of TSID

### DISCUSSION THREAD

**Discussion by xiaosuyang** [stack-of-tasks/pinocchio#2226] [Q&A]
Hello.
In the tsid robotwrapper.cpp, function RobotWrapper::frameClassicAcceleration() which is used to calculate the classic accleration of a frame:
![image](https://github.com/stack-of-tasks/pinocchio/assets/55594017/44649def-45dd-458c-8866-745359ec9699)
In row 301 and 302, a and v  are spatial acc and spatial vel of the frame f, but why they use placement.actInv(data.v[f.parent])(data.v[f.parent] is the vel of the parent joint ) instead of placement.act(). Isn't the frameXJoint.act() used to transform the vector from Joint coordinate to the frame coordinate?

**Comment by stephane-caron** [comment]
Since this question is related to TSID, you should ask it in the [TSID discussions](https://github.com/stack-of-tasks/tsid/discussions) forum.
